### DISCUSSION TITLE
Import Credit Card Transactions & Cryptocurrency Payments

### DISCUSSION THREAD

**Discussion by 9mido** [arrobalytics/django-ledger#339] [Ideas]
Credit cards are used for automatic payments as opposed to writing checks and snail mail. 

It would be nice to have some way to import business credit card transactions and track spend and maybe even categorize expenses. Even better to do it automatically. 

Cryptocurrency payments may be more popular in the future and that would be cool to track also even though that is harder since the price fluctuates.

**Comment by mnooel** [comment]
we should focus on credit card transactions for the time being. A .csv import would be useful. An API even better but I don't know if we are there yet. There is a common financial data platform called [plaid](https://plaid.com/products/auth/overview/?utm_source=google&utm_medium=search&utm_campaign=Search_G_Brand_Exact&utm_content=plaid&utm_term=plaid&utm_creative=481138286147&gclid=Cj0KCQiAzZL-BRDnARIsAPCJs70Df6n37A3xM7xtF_ArAyigyFxBdSdZdSUxEOGEkvE2LV7DLenfvhgaAk4HEALw_wcB) we could aspire to use.

Some considerations, many organizations have one or many corporate credit card accounts. For each account, they might have many cards for a set of employees. An account needs to be paid by the group but reconciled and coded by the employee. Another parent-child relationship to consider.

**Comment by 9mido** [comment]
Forgot to also add bank accounts to be connected as well besides credit cards and cryptocurrency. 

There are also different types of credit cards. The ones that have been around forever and some newer ones.

These are more advanced newer ones with unique features:
https://x1creditcard.com/
https://stripe.com/en-gb-us/corporate-card
https://stripe.com/en-gb-us/issuing

Different ways to make payments online:
https://stripe.com/en-gb-us/payments/payment-methods-guide#payment-methods-fact-sheets

Then you also have things like Paypal, Shopify, Patreon, etc.

**Comment by elarroba** [comment]
Credit card imports will be implemented on version 0.9.0
